src/server.c

```diff
@@ -1081,8 +1112,7 @@ static void on_main_dispatch(void* aml_obj)
 		process_fb_update_requests(client);
 }
 
-EXPORT
-struct nvnc* nvnc_open(const char* address, uint16_t port)
+static struct nvnc* open_common(const char *address, uint16_t port, bool unix_)
```

Comment:
Let's call that boolean `is_unix`. This is OK, but I usually try to avoid boolean arguments like this because they make code harder to read. When you come across a line in code like this:
```
open_connection(address, 0, true)
```
you have to look at the definition of the function to see what it does. However, when you read something like
```
open_connection(address, 0, SOCKTYPE_UNIX);
```
you don't really have to look it up.